VsViewer/vs_calc/CPT.py

```diff
@@ -202,7 +202,11 @@ def calc_cpt_params(self):
         for i in range(0, len(n)):
             deltan = 1
             # iterate Qtn, Ic and n until convergence
+            iteration_counter = 0
             while deltan >= 0.01:
+                iteration_counter += 1
+                if iteration_counter > 1e6:
+                    raise ValueError("CPT params calculation could not converge")
```

Comment:
Thinking we could make this a custom error to distinguish between a normal ValueError and just not converging if we wanted to use a try except around a bunch of these specific issues.